.changes/unreleased/Fixes-20240531-113620.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Fixes
+body: Remove group_federation when using serverless for iam authentication
```

Comment:
I could take it or leave it, but this might read better to an end user as:
```
Support IAM Role authentication for Redshift Serverless
```
What's here is _how_ we did it versus _what_ the user gets.

Reply:
Done!
You would think I know that...